src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,51 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+import qna.CannotDeleteException;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question")
+    private List<Answer> answers = new ArrayList<Answer>();
+    
+    protected Answers() {
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+    
+    public DeleteHistories delete() {
+        List<DeleteHistory> deleteHistories = new ArrayList<DeleteHistory>();
+        answers.forEach(answer -> deleteHistories.add(answer.delete()));
+        return DeleteHistories.fromAnswers(deleteHistories);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+    
+    public void checkDeletableByQuestion(User writer) throws CannotDeleteException {
```

Comment:
RuntimeException으로 변경해주셨다면 메서드에 예외 선언을 해주지 않아도 괜찮습니다 :)

```suggestion
    public void checkDeletableByQuestion(User writer) {
```

Reply:
그렇네요!! 꼼꼼히 봐주셔서 감사합니다!! 😁👍🏻